gradle.properties

```diff
@@ -24,5 +24,4 @@ POM_DEVELOPER_NAME=Kevin Galligan
 POM_DEVELOPER_ORG=Touchlab
 POM_DEVELOPER_URL=https://touchlab.co/
 
-kotlin.mpp.enableCInteropCommonization=true
-kotlin.mpp.commonizerLogLevel=info
+kotlin.js.ir.output.granularity=whole-program
```

Comment:
Needed this to solve some build failures. I think it was something going wrong in dead code elimination when building with the default per-module granularity. I think it's low-impact since we don't publish JS directly anyway, but I don't fully understand the subtleties here.